cc_proxy.go

```diff
@@ -98,15 +98,6 @@ func (p *ccProxy) register(pod Pod) ([]IOStream, error) {
 		return []IOStream{}, err
 	}
 
-	for i := 0; i < len(pod.containers); i++ {
-		ioStream, err := p.allocateIOStream()
-		if err != nil {
-			return []IOStream{}, err
-		}
-
-		ioStreams = append(ioStreams, ioStream)
-	}
-
```

Comment:
Well the reason is that we are no longer calling register from StartPod, but from StartAgent instead. Several calls to connect will provide the tokens needed. The root reason is that we cannot disconnect completely the proxy from the VM and then reconnect (in case we are doing a stop pod then start pod) because the proxy will expect a READY message from hyperstart but because hyperstart is already running at that time and because we already consumed the READY from the previous StartPod, the proxy will hang. That's why the register has to be called only once, and in that case, we have to handle the ioStreams with the connect (one for each container) instead of the register.